### PR TITLE
application_config_schema.js not being used and Application/Schema invalid syntax - Closes #3939

### LINKED ISSUE #3939: application_config_schema.js not being used and Application/Schema invalid syntax
### Expected behavior

- `application_config_schema.js` is validated in Application as `genesisBlockSchema` is.
- Suggested values in https://github.com/LiskHQ/lisk-sdk/issues/3151 are added to `application_config_schema.js`
- Schema definition is valid

### Actual behavior

- `application_config_schema.js`  is not being validated.
- schema is using `min` and `max` instead of `minimum` and `maximum` for limits which is invalid and pass validation with invalid values

### Steps to reproduce

Make changes in `application_config_schema.js` setting values that should fail validation and application will run

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

- application_config_schema.js is validated in Application as genesisBlockSchema is.
- Suggested values in #3151 are added to application_config_schema.js
- Schema definition was using `min` and `max` instead of `minimum` and `maximum`

### How did I fix it?

- application_config_schema.js is being validated.
- schema is using min and max instead of minimum and maximum for limits which is invalid and pass validation with invalid values

### How to test it?

- Build should be green
- Change `framework/test/fixtures/config/devnet/config.json` with values that will fail validation and application should not start

### Review checklist

* The PR resolves #3939
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/controller/application.js [hunk 1]
```diff
@@ -29,7 +29,11 @@ const Controller = require('./controller');
 const version = require('../version');
 const validator = require('./validator');
 const configurator = require('./default_configurator');
-const { genesisBlockSchema, constantsSchema } = require('./schema');
+const {
+	genesisBlockSchema,
+	constantsSchema,
+	applicationConfigSchema,
```
Comment (nazarhussain) [inline framework/src/controller/application.js]: @pablitovicente If that's the only issue, this config schema is already been loaded to configurator, you don't need to do it here.

Comment (pablitovicente) [reply framework/src/controller/application.js]: Hmm will check that code then as it was not working. It was possible to specify invalid values but maybe due to the combination of wrong schema definition... will double check.

Comment (pablitovicente) [reply framework/src/controller/application.js]: Removed application schema validation from Application. Schema fix was enough. Resolving.

File: framework/src/controller/schema/application_config_schema.js [hunk 2]
```diff
@@ -159,12 +164,12 @@ module.exports = {
 								},
 								OFFSET: {
 									type: 'integer',
-									min: 1,
+									minimum: 1,
```
Comment (nazarhussain) [inline framework/src/controller/schema/application_config_schema.js]: For future reference, here is the specification for this change. 

https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.2.4

**Pull request merged**